docs/QbeastTable.md

```diff
@@ -47,6 +53,25 @@ qbeastTable.optimize()
 qbeastTable.optimize(Seq("file1", "file2"))
 ```
 
+### Optimization of Unindexed Files
+
+There are some use cases in which a Table could have several **Unindexed Files**.
+- **Staging Data**: Enabling the Staging Area gives the possibility to **ingest data without indexing it**. Since very small appends could produce overhead during the write process, the new data would be commited to the table without reorganization. Every time the staging are size is reached, the data is indexed using the latest state of the Table. 
+- **Table Converted To Qbeast**: An existing `parquet` or `delta` Table can be converted to a `qbeast` Table through the `ConvertToQbeastCommand`. Since the table can be very big, the conversion only adds a metadata commit to the Log, indicating that from that point onwards the appends would be indexed with Qbeast.
+- **External Table Writers**: External writers can write data to the table in the underlying format (delta, hudi or iceberg)
+
+All the sets of Unindexed Files are mapped to a revision number 0. For manually indexing these files, you can use the `optimize` method with the `revisionId` parameter set to 0.
```

Comment:
All ~~the sets of~~ Unindexed Files are mapped to a revision number 0. For manually indexing these files, you can use the `optimize` method with the `revisionId` parameter set to 0.

Reply:
done in 8cab2b8